mutual-information-feature-selection/README.rst

```diff
@@ -0,0 +1,81 @@
+Demo of the MIQUBO Method of Feature Selection
+==============================================
+
+Statistical and machine-learning models use a set of input variables (features)
+to predict output variables of interest. `Feature selection`_, which can be
+part of the model design process, simplifies the model and reduces dimensionality by selecting,
+from a given set of potential features, a subset of highly informative ones. One
+statistical criterion that can guide this selection is `mutual information`_ (MI).
+
+Ideally, to select the k most relevant features, you might maximize I(Xs; Y),
+the MI between a set of k features, Xs, and the variable of interest, Y.
+This is a hard calculation because the number of states is exponential with k.
```

Comment:
I believe this means that the number of states in P(Xk,Y) is Y*(Prod_i^k dim(X_i) ) where dim is the dimension of the feature. For binary features this would be 2^k+1.